src/box/box.cc

```diff
@@ -3436,12 +3436,17 @@ bootstrap_master(const struct tt_uuid *replicaset_uuid)
  * @param[out] start_vclock  the vector time of the master
  *                           at the moment of replica bootstrap
  */
-static void
+static int
```

Comment:
Please, describe the return value in the comment.

The function might throw an exception on an unrecoverable error, it might return -1 in case of a transient error and it might return 0 in case everything is fine.

Reply:
Comment is below